wgpu-hal/src/vulkan/command.rs

```diff
@@ -427,6 +427,11 @@ impl crate::CommandEncoder<super::Api> for super::CommandEncoder {
             vk_info = vk_info.push_next(&mut vk_attachment_info);
         }
 
+        if let Some(label) = desc.label {
+            self.begin_debug_marker(label);
+            self.rpass_debug_marker_active = true;
```

Comment:
we need to reset this properly somewhere. Otherwise, we can be in a situation where the recording is discarded, but it would think that the marker is active on the next command buffer

Reply:
That work?